### ISSUE TITLE
Why we are setting ignoreLineBreaks to true by default

### ISSUE THREAD

**Issue by complanboy2** [onelogin/java-saml#213]
Here, https://github.com/onelogin/java-saml/blob/master/core/src/main/java/com/onelogin/saml2/util/Util.java#L116

we are setting the system property "org.apache.xml.security.ignoreLineBreaks" to true by default which will instruct xmlutils to ignore line breaks.

Is there a way that we can override this value dynamically ! 
How one should say to NOT ignore line breaks in the signature verification ?

**Comment by pitbulk** [comment]
"org.apache.xml.security.ignoreLineBreaks" = true will make that XMLUtils don't build XML with linebreaks, but it does not affect the verification process.

Most of Java SAML implementations use it:
* [opensaml](https://www.programcreek.com/java-api-examples/?code=apigee/java-opensaml2/java-opensaml2-master/src/main/java/org/opensaml/DefaultBootstrap.java)
* [apache/wss4j](https://github.com/apache/wss4j/blob/649447d68352ef8d93d873eaafec6c58d2ea1a81/ws-security-common/src/main/java/org/apache/wss4j/common/crypto/WSProviderConfig.java#L169)
* [eidas](https://github.com/france-connect/eidas-identity-provider/blob/b9b6930137e2e2fa4bb20fd2714deee0fb7bbd94/EIDAS-Encryption/src/main/lib/org/opensaml/opensaml/2.6.5-eidas_1/src/main/java/org/opensaml/DefaultBootstrap.java#L190)
* [seam/security](https://github.com/seam/security/blob/8fda0bd94b02d62074d8e340cef340fe61b2dca2/external/src/main/java/org/jboss/seam/security/external/saml/SamlSignatureUtilForPostBinding.java#L52)
* [apache/camel](https://github.com/apache/camel/blob/d0e04616cebee03ecdd5ed5333ed57a93237b62c/components/camel-xmlsecurity/src/main/java/org/apache/camel/component/xmlsecurity/processor/SantuarioUtil.java#L44)

**Comment by complanboy2** [comment]
Right, this is to tell XMLUtils to not include the line breaks. Libraries mentioned except seam are reading from system property and setting if it's null. With that we have an option to set that to false.

But in our OneLogin, we are directly setting the property which leave no room where we don't want to ignore line breaks.

If we are able to read from the property and set or set it dynamically (I know that XMLUtils can not take dynamic values, but if there is a way that we support this dynamic flag. Eg: if this flag is set, we may ignore line breaks in attribute values by altering payload etc). that would be very much helpful.

Thank you for your reply.

**Comment by pitbulk** [comment]
I see, do you want to provide a PR with a similar solution?

**Comment by complanboy2** [comment]
Sure, I'm happy to provide a PR, thank you.

**Comment by pitbulk** [comment]
@complanboy2 are you still interested in providing such PR?
